code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -353,63 +371,45 @@
 	return
 
 /mob/living/simple_animal/bot/medbot/proc/assess_patient(mob/living/carbon/C)
-	//Time to see if they need medical help!
-	if(C.stat == 2)
-		return 0 //welp too late for them!
-
-	if(C.suiciding)
-		return 0 //Kevorkian school of robotic medical assistants.
-
-	// is secretly a silicon
-	if(ishuman(C))
-		var/mob/living/carbon/human/H = C
-		if(H.species && H.species.reagent_tag == PROCESS_SYN)
-			return 0
-
-	if(emagged == 2) //Everyone needs our medicine. (Our medicine is toxins)
-		return 1
-
-	if((skin == "bezerk") && (!("syndicate" in C.faction)))
-		return 0
-
-	if(declare_crit && C.health <= 0) //Critical condition! Call for help!
+//Check the status of the patient and return appropriate condition.
+	if((skin == "bezerk") && (!("syndicate" in C.faction))) // If we're a syndie and they aren't then fail.
```

Comment:
Bezerk (not berzerk)? Is this typo intentional?

Reply:
Yes as it's how it was originally. I could change it to the correct spelling.